### PR TITLE
Added Linux metadata

### PR DESCRIPTION
**Pull request by Mailaender**
This adds metadata based on the [AppStream](https://www.freedesktop.org/wiki/Distributions/AppStream/) standard. It is a requirement for inlusion at @flathub where I just submitted https://github.com/flathub/flathub/pull/4472 and where you are of course invited.

### CODE REVIEW

**Comment by nextghost** [comment]
Sorry for the long delay. If you'd like a higher resolution icon, here's one made from the startup logo animation:
icon

File: signus/freedesktop/io.github.signus_game.signus.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.0-->
+  <id>io.github.signus_game.signus</id>
+  <name>Signus: The Artefact Wars</name>
+  <summary>turn-based strategy game</summary>
+  <developer_name></developer_name>
```
Comment (nextghost) [inline signus/freedesktop/io.github.signus_game.signus.metainfo.xml:7]: Developer could be either `Valacirca` (the company that made the original commercial version) or `Signus game team`.

File: signus/freedesktop/io.github.signus_game.signus.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <description>
+    <p>Signus is a Czech turn-based strategy game from 1998, developed by Valacirca and released by Zima Software. Since 2002 it is free and open source software with both code and assets.</p>
+    <p>The game features 40 units: scout vehicles, tanks, artillery, helicopters, planes, ships, and support units providing repairs, refuels and ammo. There are also nine buildings and five defense towers in the game.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://signus.paan.cz/wp-content/uploads/2013/03/mise7-1.jpg</image>
```
Comment (nextghost) [inline signus/freedesktop/io.github.signus_game.signus.metainfo.xml:16]: This screenshot shows the original commercial version of the game. The open version uses different fonts.

File: signus/freedesktop/io.github.signus_game.signus.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="1.96.0" date="2021-05-28" type="stable">
+      <url>https://github.com/signus-game/signus/releases/tag/v1.96.0</url>
+    </release>
+  </releases>
+  <url type="homepage">https://signus.paan.cz/en/</url>
```
Comment (nextghost) [inline signus/freedesktop/io.github.signus_game.signus.metainfo.xml:24]: signus.paan.cz is a fan website unrelated to this project. Signus currently does not have any homepage other than the Github repo.

**Review by nextghost — CHANGES_REQUESTED** [review]

File: signus/freedesktop/io.github.signus_game.signus.metainfo.xml (hunk 3 above)
Comment (Mailaender) [reply signus/freedesktop/io.github.signus_game.signus.metainfo.xml:24]: A website is required for @flathub. Which should I use instead?

File: signus/freedesktop/io.github.signus_game.signus.metainfo.xml (hunk 2 above)
Comment (Mailaender) [reply signus/freedesktop/io.github.signus_game.signus.metainfo.xml:16]: Can you point me to a URL of an alternative?

Comment (nextghost) [reply signus/freedesktop/io.github.signus_game.signus.metainfo.xml]: I've tweeted a few screenshots during development:
https://twitter.com/next_ghost_cz/status/1398376572277624842
https://twitter.com/next_ghost_cz/status/1325556150562136064

Or there's a bunch of good screenshots in this article but I don't know how much traffic the hosting can take:
https://www.high-voltage.cz/2021/signus-the-artefact-wars-zapomenuta-klasika-valecnych-tahovek/

Please don't use the repo picture for social media, it's horribly distorted by GitHub.

Comment (Mailaender) [reply signus/freedesktop/io.github.signus_game.signus.metainfo.xml]: The screenshots will be mirrored. I will take the English language Twitter ones.

**Review by nextghost — APPROVED** [review]

**Pull request merged**